Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -21,6 +21,7 @@ import SKSupport
 import TSCBasic
 import TSCLibc
 import TSCUtility
+import PackageLoading
```

Comment:
It would be nice if we could contain all imports of SwiftPM modules to the SwiftPM build system file to keep things contained.  No need to change it in this PR though.